addons/ai_apps/static/src/message_patch.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Message" t-inherit-mode="extension">
+        <xpath expr="//*[hasclass('o-mail-Message-contentContainer')]" position="attributes">
+            <attribute name="t-attf-class" add="{{ props.thread.channel_type === 'ai_composer' ? 'flex-column' : '' }}" separator=" "/>
+        </xpath>
+    </t>
+    <t t-inherit="mail.Message.actions" t-inherit-mode="extension">
+        <xpath expr="//button" position="attributes">
+            <attribute name="class" remove="border-0" separator=" "/>
+            <attribute name="t-attf-class" add="{{ action.id === 'insertToComposer' ? 'rounded-3 border-2' : 'border-0' }}" separator=" "/>
+            <attribute name="t-attf-style" add="{{ action.id === 'insertToComposer' ? 'background-color: lightgray; border-color: gray;' : '' }}"/>
```

Comment:
You may want to use `text-100`, `text-200`, etc. and `bg-100`, `bg-200` etc. rathen than `lightgray`/`gray` which I think changes across browsers, or at least doesn't fit well with the themes at hand (yes, it should look nice in both white and dark theme, the classname above adapt with the theme).